routes/web.php

```diff
@@ -17,10 +17,17 @@
     return view('welcome');
 });
 
-Route::get('/test', function () {
-    return 'oke';
+Route::get('/about', function () {
+    $name = filter_input(INPUT_GET, 'name', FILTER_SANITIZE_STRING);
+    return view('about', ['name' => $name]);
 });
 
-Route::any('/test', function () {
-    return 'oke';
+Route::get('/contact', function () {
+    // check is phone number is valid
+    $phone = filter_input(INPUT_GET, 'phone', FILTER_SANITIZE_STRING);
+    if (preg_match('/^[0-9]{3}-[0-9]{3}-[0-9]{4}$/', $phone)) {
+        return true;
+    } else {
+        return false;
+    }
 });
```

Comment:
the code review..
First, let's look at the Route::get('/about', function () { line. We can see that it is using a filter_input() function to sanitize the input. This is good practice as it helps protect against malicious user input.

Next, let's look at the Route::get('/contact', function () { line. Here we can see that the code is using a preg_match() function to check if the phone number is valid. This is also good practice as it helps ensure that the user's input is valid and will not cause any errors.

Overall, the code looks good and is following best practices for handling user input.